core/domain/platform_parameter_registry_test.py

```diff
@@ -162,21 +162,27 @@ def test_update_parameter(self):
         parameter_name = 'parameter_a'
         self._create_example_parameter_with_name(parameter_name)
 
+        new_rule_dicts = [
+            {
```

Comment:
Just a note, you can avoid making this look so expansive by doing the following:

```
        new_rule_dicts = [{
            'filters': [{
                'type': 'server_mode',
                'conditions': [('=', FEATURE_STAGES.dev.value)]
            }],
            'value_when_matched': 'updated'
        }]
```

Ditto elsewhere.

Reply:
Thank you @seanlip this helps to reduce extra lines to a much grater extent.

Reply:
Thank you for review, will take care of these points while creating a separate PR.